### DISCUSSION TITLE
Welcome to unitree_guide Discussions!

### DISCUSSION THREAD

**Discussion by xiaoliangstd** [unitreerobotics/unitree_guide#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Harbin-cell** [comment]
unitree_guide中gazebo正常启动后unitree_guide junior_ctrl也正常执行后机器狗没反应，并且ros topic list没有输出有关关节信息的话题

**Reply by clmm233** [reply]
我的也是呢哥 你解决了没 启动gazebo之后 输入按键 控制器终端会输出反馈信息 但是狗根本不动 找不到原因

**Comment by Harbin-cell** [comment]
你是不是在狗上加东西了，还有电脑配置也有关系
然后你在其他电脑上试试对比一下









---Original---
From: "xilai ***@***.***&gt;
Date: Thu, Jul 31, 2025 16:13 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [unitreerobotics/unitree_guide] Welcome to unitree_guideDiscussions! (Discussion #22)




 
我的也是呢哥 你解决了没 启动gazebo之后 输入按键 控制器终端会输出反馈信息 但是狗根本不动 找不到原因
